apps/admin-gui/src/app/shared/components/dialogs/remove-members-dialog/remove-members-dialog.component.html

```diff
@@ -30,7 +30,7 @@ <h1 mat-dialog-title>{{'DIALOGS.REMOVE_MEMBERS.TITLE' | translate}}</h1>
       mat-flat-button
       class="ml-auto"
       (click)="onCancel()">
-      {{'DIALOGS.DELETE_GROUP.CANCEL' | translate}}
+      {{'DIALOGS.REMOVE_MEMBERS.CANCEL' | translate}}
```

Comment:
For such bug fixes, which are not related to the main purpose of a  PR, you should create a separate PR.